lib/fastlane/plugin/ddg_apple_automation/actions/tag_release_action.rb

```diff
@@ -22,12 +23,36 @@ def self.setup_constants(platform)
         end
       end
 
+      # rubocop:disable Metrics/PerceivedComplexity
       def self.run(params)
         platform = params[:platform] || Actions.lane_context[Actions::SharedValues::PLATFORM_NAME]
         Helper::GitHelper.setup_git_user
 
         setup_constants(platform)
 
+        unless params[:is_prerelease]
+          branch = other_action.git_branch
+          begin
+            Helper::GitHelper.unfreeze_release_branch(branch, platform, params[:github_token])
+          rescue StandardError => e
+            Helper::GitHubActionsHelper.set_output("stop_workflow", true)
+            Helper::DdgAppleAutomationHelper.report_error(e)
+            UI.important("Failed to unfreeze release branch. Cannot proceed with the public release. Please unfreeze manually and run the workflow again.")
+            task_id = AsanaExtractTaskIdAction.run(task_url: params[:asana_task_url])
+            AsanaReportFailedWorkflowAction.run(
```

Comment:
Reusing “Workflow Failed” notification when unfreezing (i.e. deleting a draft release) fails, that would post a comment directly to the release task.